cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -174,20 +175,14 @@ private static ImagePullPolicy parseImagePullPolicy(String imagePullPolicyEnvVar
                 ModelUtils.parseImageMap(connectImages),
                 ModelUtils.parseImageMap(connectS2IImages),
                 ModelUtils.parseImageMap(mirrorMakerImages));
-
-        for (String version : lookup.supportedVersions()) {
-            if (lookup.kafkaImage(null, version) == null) {
-                LOGGER.warn("{} does not provide an image for version {}", STRIMZI_KAFKA_IMAGES, version);
-            }
-            if (lookup.kafkaConnectVersion(null, version) == null) {
-                LOGGER.warn("{} does not provide an image for version {}", STRIMZI_KAFKA_CONNECT_IMAGES, version);
-            }
-            // Need to know whether we're on OS to decide whether to valid s2i
-            if (lookup.kafkaMirrorMakerImage(null, version) == null) {
-                LOGGER.warn("{} does not provide an image for version {}", STRIMZI_KAFKA_MIRROR_MAKER_IMAGES, version);
-            }
+        try {
+            lookup.validateKafkaImages(lookup.supportedVersions());
+            lookup.validateKafkaConnectImages(lookup.supportedVersions());
+            // TODO Need to know whether we're on OS to decide whether to valid s2i
```

Comment:
Is this something what should be implemented in this PR? Or in some other PR? If it is other PR, how is it validated in the meantime?

Reply:
I don't think it should be addressed in this PR (it was a problem before, I just changed a `//` comment into a `// TODO` comment). I do think it should be addressed in some future PR. In the meantime it is not validation, just as previously.

Reply:
But you seem to have added the method `validateKafkaConnectS2IImages`. Why not just call it?